DependencyInjection/SonataCoreExtension.php

```diff
@@ -57,6 +57,7 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('twig.xml');
         $loader->load('model_adapter.xml');
         $loader->load('core.xml');
+        $loader->load('utils.xml');
```

Comment:
This is most of the time a bad idea to have a "utils" package/namespace/directory/...